lib/lib.js

```diff
@@ -74,7 +74,12 @@ function maybeTestForPlusOne(config, comment) {
   return /\+1/.test(comment);
 }
 
+exports.formatReviewedByMetaData = formatReviewedByMetaData;
 function formatReviewedByMetaData(reviewer) {
+  if (!reviewer.email) {
```

Comment:
I would suggest not duplicating the email format string/format operation and maybe rewrite it like following:
```
assert(typeof (reviewer.name) === 'string');
var formattedComponents = [];
var formattedEmail;

formattedComponents.push(util.format('%s', reviewer.name));

if (reviewer.email) {
  formattedComponents.push(util.format('<%s>', reviewer.email));
}

return formattedComponents.join(' ').trim();
```
or something along those lines.